### DISCUSSION TITLE
Rebind hand gestures

### DISCUSSION THREAD

**Discussion by Spartanets174** [alvr-org/ALVR#2179] [General]
![Снимок экрана 2024-06-07 102131](https://github.com/alvr-org/ALVR/assets/99803949/32e003d6-1122-479b-b65f-3a03c5955b6a)
For example I want to bind grip button through curling all 5 fingers, or trigger on on thumb

**Comment by zmerp** [comment]
This is planned. We have the button mapping system in place for controllers, we just need to add finger curls and pinches as base inputs.

**Reply by Spartanets174** [reply]
Oh that's really cool. Can you tell which version this could possibly be in?

**Reply by zmerp** [reply]
There is no ETA for now, I'm dealing with other changes.
